gramps/plugins/rel/rel_nl.py

```diff
@@ -69,9 +70,207 @@
+                   "overoudooms/tantes 41e graad",
+                   "overoudooms/tantes 42e graad",
+                   "overoudooms/tantes 43e graad",
+                   "overoudooms/tantes 44e graad",
+                   "overoudooms/tantes 45e graad",
+                   "overoudooms/tantes 46e graad",
+                   "overoudooms/tantes 47e graad",
+                   "overoudooms/tantes 48e graad",
+                   "overoudooms/tantes 49e graad",
+                   "overoudooms/tantes 50e graad", ]
+
+_children_level = ["",
+                   "kinderen",
+                   "kleinkinderen",
+                   "achterkleinkinderen",
+                   "betachterkleinkinderen",
+                   "kleinkinderen 5e graad",
+                   "kleinkinderen 6e graad",
+                   "kleinkinderen 7e graad",
+                   "kleinkinderen 8e graad",
+                   "kleinkinderen 9e graad",
+                   "kleinkinderen 10e graad",
+                   "kleinkinderen 11e graad",
+                   "kleinkinderen 12e graad",
+                   "kleinkinderen 13e graad",
+                   "kleinkinderen 14e graad",
+                   "kleinkinderen 15e graad",
+                   "kleinkinderen 16e graad",
+                   "kleinkinderen 17e graad",
+                   "kleinkinderen 18e graad",
+                   "kleinkinderen 19e graad",
+                   "kleinkinderen 20e graad",
+                   "kleinkinderen 21e graad",
+                   "kleinkinderen 22e graad",
+                   "kleinkinderen 23e graad",
+                   "kleinkinderen 24e graad",
+                   "kleinkinderen 25e graad",
+                   "kleinkinderen 26e graad",
+                   "kleinkinderen 27e graad",
+                   "kleinkinderen 28e graad",
+                   "kleinkinderen 29e graad",
+                   "kleinkinderen 30e graad",
+                   "kleinkinderen 31e graad",
+                   "kleinkinderen 32e graad",
+                   "kleinkinderen 33e graad",
+                   "kleinkinderen 34e graad",
+                   "kleinkinderen 35e graad",
+                   "kleinkinderen 36e graad",
+                   "kleinkinderen 37e graad",
+                   "kleinkinderen 38e graad",
+                   "kleinkinderen 39e graad",
+                   "kleinkinderen 40e graad",
+                   "kleinkinderen 41e graad",
+                   "kleinkinderen 42e graad",
+                   "kleinkinderen 43e graad",
+                   "kleinkinderen 44e graad",
+                   "kleinkinderen 45e graad",
+                   "kleinkinderen 46e graad",
+                   "kleinkinderen 47e graad",
+                   "kleinkinderen 48e graad",
+                   "kleinkinderen 49e graad",
+                   "kleinkinderen 50e graad", ]
+
+_nephews_nieces_level = ["",
+                         "broers en zussen",
+                         "neven/nichten",
```

Comment:
No "en" instead of a slash, like with "broers en zussen"?

Reply:
Yes, good idea. Let's change it into:
"neven en nichten",

Reply:
Do you want the whole list changed in the same way?